src/components/navbar/navbar.tsx

```diff
@@ -0,0 +1,21 @@
+import Logo from '../logo/logo';
+import Button from '../reusable/buttons';
+function navbar() {
+  return (
+    <nav className="px-5 flex justify-between items-center">
+      <Logo />
+      <ul className='flex list-none'>
+        <li className='mx-5 cursor-pointer text-base font-medium font-poppins tracking-wider hover:text-purple-600'>Home</li>
```

Comment:
make an array with these values. `['Market Place', 'Notice Board', 'Academics',]` and then run ` map()` on this and return `<li></li>`

Reply:
right done